src/components/app/details/cicdHistory/TriggerDetails.tsx

```diff
@@ -145,6 +158,20 @@ const ProgressingStatus = React.memo(
         } else if (stage !== 'DEPLOY') {
             abort = () => cancelPrePostCdTrigger(pipelineId, triggerId)
         }
+
+        useEffect(() => {
+            setDurationStr(formatDurationDiff(startedOn, Date()))
```

Comment:
Please for built in formatter to formate time

Reply:
### Yes the built in formatter was not formatting time in the way that we wanted it to be displayed, so I created my own formatter, Sohel  checked the code and told me to do so.

Reply:
Yes the built in formatter was not formatting time in the way that we wanted it to be displayed, so I created my own formatter, Sohel  checked the code and told me to do so.